companies/digitalo.json

```diff
@@ -0,0 +1,25 @@
+{
+    "slug": "digitalo",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Re-In Retail International GmbH",
+    "runs": [
+        "Re-In Retail International GmbH",
+        "Digitalo",
```

Comment:
Just "digitalo.de" is fine, otherwise the name will appear multiple times on the website.